lib/fizzy/parser_expr.cpp

```diff
@@ -255,6 +276,7 @@ parser_result<Code> parse_expr(const uint8_t* pos, const uint8_t* end, bool have
             uint8_t arity;
             std::tie(arity, pos) = parse_blocktype(pos, end);
             code.immediates.push_back(arity);
+            frame.stack_height += arity;
```

Comment:
A bit confusing, maybe add a comment that it's for enclosing frame, something like "outer frame will have additional item on stack after exit from block with result"

Reply:
Added more detailed comments in all instances.